lib/pgBackRest/Info.pm

```diff
@@ -134,25 +134,37 @@ sub process
+            foreach my $oBackupInfo (@{$$oStanzaInfo{&INFO_BACKUP_SECTION_BACKUP}})
             {
-                my $oOldestBackup = $$oStanzaInfo{&INFO_BACKUP_SECTION_BACKUP}[0];
-
-                $strOutput .= '    oldest backup label: ' . $$oOldestBackup{&INFO_KEY_LABEL} . "\n";
-                $strOutput .= '    oldest backup timestamp: ' .
-                              timestampFormat(undef, $$oOldestBackup{&INFO_SECTION_TIMESTAMP}{&INFO_KEY_START}) . "\n";
-
-                my $oLatestBackup = $$oStanzaInfo{&INFO_BACKUP_SECTION_BACKUP}[@{$$oStanzaInfo{&INFO_BACKUP_SECTION_BACKUP}} - 1];
-
-                $strOutput .= '    latest backup label: ' . $$oLatestBackup{&INFO_KEY_LABEL} . "\n";
-                $strOutput .= '    latest backup timestamp: ' .
-                              timestampFormat(undef, $$oLatestBackup{&INFO_SECTION_TIMESTAMP}{&INFO_KEY_START}) . "\n";
+                $strOutput .= '    '.$$oBackupInfo{&INFO_KEY_TYPE} .' backup: '. $$oBackupInfo{&INFO_KEY_LABEL} . "\n";
+                $strOutput .= '        start / stop timestamp: ' .
+                                        timestampFormat(undef, $$oBackupInfo{&INFO_SECTION_TIMESTAMP}{&INFO_KEY_START}) .
+                                        ' / ' .
+                                        timestampFormat(undef, $$oBackupInfo{&INFO_SECTION_TIMESTAMP}{&INFO_KEY_STOP}) . "\n";
+                $strOutput .= '        database size: ' .
+                                        (defined($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_KEY_SIZE}) ?
+                                        fileSizeFormat($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_KEY_SIZE}) : '') .
+                                        (', backup size: ' .
+                                        (defined($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_KEY_DELTA}) ?
+                                        fileSizeFormat($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_KEY_DELTA}) : '')) . "\n";
+                $strOutput .= '        repository size: ' .
+                                        (defined($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_SECTION_REPO}{&INFO_KEY_SIZE}) ?
+                                        fileSizeFormat($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_SECTION_REPO}{&INFO_KEY_SIZE}) : '').
+                                        (', repository backup size: ' .
+                                        (defined($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_SECTION_REPO}{&INFO_KEY_DELTA}) ?
+                                        fileSizeFormat($$oBackupInfo{&INFO_SECTION_INFO}{&INFO_SECTION_REPO}{&INFO_KEY_DELTA}) : '')) . "\n";
+
+                # list the backup reference chain, if any, for this back up
+                if (defined($$oBackupInfo{&INFO_KEY_REFERENCE}))
+                {
+                    $strOutput .= '        backup reference list: ' . (join(', ' ,@{$$oBackupInfo{&INFO_KEY_REFERENCE}})) . "\n";
+                }
+                $strOutput .= "\n";
             }
```

Comment:
I've modified the formatting / spacing in this code: https://github.com/pgbackrest/pgbackrest/commit/e55913879d01bdb01c9f62783c789a39a000e485